### ISSUE TITLE
Python3.7 compatibility: async is a reserved keyword

### ISSUE THREAD

**Issue by justintwr** [Isilon/isilon_sdk#72]
While trying to import the library with python3.7, the following error occurs:
```
    import isi_sdk_8_1_0
  File "C:\Users\ricej4\AppData\Roaming\Python\Python37\site-packages\isi_sdk_8_1_0\__init__.py", line 19, in <module>
    from isi_sdk_8_1_0.api.antivirus_api import AntivirusApi
  File "C:\Users\ricej4\AppData\Roaming\Python\Python37\site-packages\isi_sdk_8_1_0\api\__init__.py", line 6, in <module>
    from isi_sdk_8_1_0.api.antivirus_api import AntivirusApi
  File "C:\Users\ricej4\AppData\Roaming\Python\Python37\site-packages\isi_sdk_8_1_0\api\antivirus_api.py", line 129
    async=params.get('async'),
        ^
SyntaxError: invalid syntax
```

**Comment by Atomicology** [comment]
This issue has been filed here as well: https://github.com/Isilon/isilon_sdk_python/issues/9

**Comment by nalinkanwar** [comment]
Fixed in Isilon SDK v0.2.11.

NOTE: due to the nature of the fix, any existing scripts which were using the 'async' keyword needs to be updated. This keyword has been replaced by 'async_req' now.
